CMakeLists.txt

```diff
@@ -2406,6 +2407,9 @@ if(SDL_SHARED)
     set_property(TARGET SDL2 APPEND_STRING PROPERTY COMPILE_FLAGS "-fobjc-arc")
     target_compile_definitions(SDL2 PRIVATE IOS_DYLIB=1)
   endif()
+  if(WAYLAND_LIBDECOR)
+    target_include_directories(SDL2 PRIVATE "${libdeocr_INCLUDE_DIRS}")
```

Comment:
Just realized this and the line further down has `libdeocr` rather than `libdecor` - could this be why we need the "libdecor-0.1" folder in the `#include`s? From what I can tell the pkg-config for libdecor already includes it.

Reply:
I had `libdecor-0.1/` in the `#include` because I did not realise that I was actually adding the include dir in CMake.

I just realised now that there is a typo in the CMake variable. This is fixed now and including `libdecor.h` is enought.